sci-libs/vtk/vtk-9.0.1.ebuild

```diff
@@ -294,7 +294,25 @@ src_configure() {
 	fi
 
 	if use cuda; then
-		mycmakeargs+=( -DVTKm_CUDA_Architecture="native" )
+		local MY_CUDA_ARCH=native
+		if [[ -n ${VTK_CUDA_ARCH} ]]; then
+			# we ignore fermi arch, because current nvidia-cuda-toolkit-11*
+			# no longer supports it
+			case ${VTK_CUDA_ARCH} in
+				kepler|maxwell|pascal|volta|turing|ampere|all|none)
+					MY_CUDA_ARCH=${VTK_CUDA_ARCH}
+					;;
+				*)
+					# use default value of MY_CUDA_ARCH, ignore any other values
+					;;
+			esac
+		fi
+		ewarn "Using CUDA architecture ${MY_CUDA_ARCH}"
+		ewarn "If auto detection fails for you, please try and export the"
+		ewarn "VTK_CUDA_ARCH environment variable to one of the common arch"
+		ewarn "names"
```

Comment:
Think users may potentially not know what this refers to nvidia's naming schemes, maybe an example arch or two? The displayed "native" default doesn't say much about what's expected here.